### PR TITLE
Fix: Feedback Button Present on Each Page and Persistent

### PR DESCRIPTION
**Pull request by FlyteWizard**
# Overview

The Feedback Button is now persistent as it remains on the main index page for the application. Fixed some accessibility issues (colour + image alt attribute). Updated language for feedback prompt.

## Close Up on Feedback Button
Capture d’écran, le 2021-07-11 à 11 55 17

## Overview of Feedback Button
Capture d’écran, le 2021-07-11 à 11 55 24

## Old Feedback Button
Capture d’écran, le 2021-07-11 à 12 07 43

## General

Let me know if you find anything breaking, or would like to see something updated/fixed.

### CODE REVIEW

File: src/common/sidebar/containers/SidebarTemplate.tsx [hunk 1]
```diff
@@ -56,9 +55,6 @@ export function SidebarTemplate({ children, term, title }: PropsWithChildren<Sid
         <Flex minW="80%" overflow="auto" justifyContent="center" boxShadow="lg" zIndex={56}>
           {children}
         </Flex>
-        <Box pos="absolute" bottom="0" right="0" zIndex={999} p={25}>
```
Comment (keithradford) [inline src/common/sidebar/containers/SidebarTemplate.tsx:59]: `Box` is still in the imports

**Review by keithradford — APPROVED** [review]
LGTM, with one nit!

File: src/common/sidebar/containers/SidebarTemplate.tsx (hunk 1 above)
Comment (FlyteWizard) [reply src/common/sidebar/containers/SidebarTemplate.tsx:59]: Ahaha, I swear I checked all of them this time. We should really catch this in the linter. Should I make an issue?

Comment (keithradford) [reply src/common/sidebar/containers/SidebarTemplate.tsx:59]: Yeah, let's make an issue and I'll address it ASAP. This is definitely the most common warning and we've been meaning to fix it for a while but always seem to forget 😅

**Pull request merged**